arangod/Aql/ClusterBlocks.h

```diff
@@ -51,20 +51,19 @@ class AqlItemBlock;
 struct Collection;
 class ExecutionEngine;
 
-class BlockWithClients : public ExecutionBlock {
+class ClusterBlocks : public ExecutionBlock {
```

Comment:
Actually my suggestion to rename the cluster blocks was primarily meant to rename the files `ClusterBlocks.h / .cpp`, as they do not contain the cluster blocks anymore. I think the best would be to rename both file and class either to `BlockWithClients`, or to something better (the best I currently come up with is `ClusterBlockBase`).